Automattic/jetpack
# Custom Post Types: Changing Nova methods to be public

This PR should fix #25077

#### Changes proposed in this Pull Request:

* All private functions have been changed to public.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* I've not been able to replicate issues and therefore check fixes as of yet, however testing the [Auberge theme](https://en-gb.wordpress.org/themes/auberge/) to make sure any of the extended functions work as expected would be the best way to test this.
